### PR TITLE
Update gsg-k8s Docs to include simplified steps with pod & DNS names.

### PR DESCRIPTION
**Pull request by techcet**
Signed-off-by: Cynthia Thomas <cynthia@covalent.io>

```release-note
Use DNS names in getting started guides
```

### CODE REVIEW

File: examples/minikube/demo.yaml [hunk 1]
```diff
@@ -26,35 +26,27 @@ spec:
...
+  labels:
+    id: app2
 spec:
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        id: app2
-    spec:
       containers:
```
Comment (tgraf) [inline examples/minikube/demo.yaml]: The indentation is off now. I'm surprised that this actually works.

**Review by tgraf — CHANGES_REQUESTED** [review]

**Review by aalemayhu — APPROVED** [review]

File: examples/minikube/demo.yaml (hunk 1 above)
Comment (techcet) [reply examples/minikube/demo.yaml]: It worked before, but I tightened up the indentation anyhow.

**Comment by tgraf** [comment]
Rebased and squashed

**Review by ianvernon — APPROVED** [review]

**Comment by raybejjani** [comment]
rebased

**Comment by manalibhutiyani** [comment]
Rebased.

**Comment by manalibhutiyani** [comment]
All review comments are addressed. Waiting for jenkins and will merge after that.

**Review by manalibhutiyani — APPROVED** [review]

**Pull request merged**